app/robots.ts

```diff
@@ -0,0 +1,14 @@
+import { MetadataRoute } from 'next'
+
+export default function robots(): MetadataRoute.Robots {
+  const baseUrl = process.env.NEXTAUTH_URL || 'https://cofe.ai'
```

Comment:
Why we should use the `NEXTAUTH_URL`? I don't see have this env anywhere else, and actually my domain is blog.minghe.me for my app